<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonOptionsPage</class>
 <widget class="QWidget" name="CommonOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>357</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="behaviorBox">
     <property name="title">
      <string>GUI Behavior</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxUseAlternatingRowColors">
        <property name="text">
         <string>Use alternating row colors in debug views</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxUseToolTipsInMainEditor">
        <property name="text">
         <string>Use tooltips in main editor while debugging</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxListSourceFiles">
        <property name="toolTip">
         <string>Checking this will populate the source file view automatically but might slow down debugger startup considerably.</string>
        </property>
        <property name="text">
         <string>Populate source file view automatically</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxRegisterForPostMortem">
        <property name="toolTip">
         <string>Register Qt Creator for debugging crashed applications.</string>
        </property>
        <property name="text">
         <string>Use Qt Creator for post-mortem debugging</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxCloseBuffersOnExit">
        <property name="toolTip">
         <string>Close temporary buffers on debugger exit.</string>
        </property>
        <property name="text">
         <string>Close temporary buffers on debugger exit</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxSwitchModeOnExit">
        <property name="toolTip">
         <string>Switch to previous mode on debugger exit.</string>
        </property>
        <property name="text">
         <string>Switch to previous mode on debugger exit</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelMaximalStackDepth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Maximal stack depth:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="spinBoxMaximalStackDepth">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="specialValueText">
           <string>&lt;unlimited&gt;</string>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="singleStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
